test/parallel/test-fs-watchfile.js

```diff
@@ -16,7 +16,7 @@ assert.throws(function() {
 
 assert.throws(function() {
   fs.watchFile(new Object(), common.noop);
-}, /Path must be a string/);
+}, common.expectsError({code: 'ERR_INVALID_ARG_TYPE', type: TypeError}));
```

Comment:
Could you add an assertion on the message as well?
(Changing the message is `semver-major` so let's enforce it)